citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ViafAuthorityImporter.java

```diff
@@ -180,6 +181,7 @@ public AuthoritySearchResult searchAuthorities(String firstName, String lastName
                     if (iterator.hasNext()) {
                         String name = iterator.next().getText();
                         authority.setName(name);
+                        authority.setSource("Viaf");
```

Comment:
see above. also, I think we have reached critical mass here to extract a method and call it from the two places instead of duplicate code.